### PR TITLE
Users already on Pro are being asked to upgrade to Pro for two Earn features

### PR DESCRIPTION
**Pull request by claudiucelfilip**
#### Changes proposed in this Pull Request

This adds the two Earn Features to the Pro plan config and updates the related card copies to mention Pro plan.

#### Testing instructions

* Go to `/earn/[site]?flags=plans/pro-plan` on a Free site
* The "Collect PayPal payments" and "Earn ad revenue" should show the `Unlock this Feature` buttons .
Screenshot on 2022-03-30 at 11-55-09

* Go to `/earn/[site]?flags=plans/pro-plan` on a Pro site
* The "Collect PayPal payments" and "Earn ad revenue" should be unlocked
Screenshot on 2022-03-30 at 12-01-22

### CODE REVIEW

**Comment by claudiucelfilip** [comment]
[Users already on Pro are being asked to upgrade to Pro for two Earn features](https://trello.com/c/mbOJv2wZ/72-users-already-on-pro-are-being-asked-to-upgrade-to-pro-for-two-earn-features)

**Review by southp — APPROVED** [review]
It works as described in the test plan. LGTM 🚢

File: packages/calypso-products/src/plans-list.tsx [hunk 1]
```diff
@@ -1619,5 +1619,7 @@ PLANS_LIST[ PLAN_WPCOM_PRO ] = {
 		FEATURE_SFTP_DATABASE,
 		FEATURE_SITE_BACKUPS_AND_RESTORE,
 		FEATURE_JETPACK_ESSENTIAL,
+		FEATURE_SIMPLE_PAYMENTS,
+		FEATURE_WORDADS_INSTANT,
```
Comment (chriskmnds) [inline packages/calypso-products/src/plans-list.tsx:1623]: Not sure why these are needed, but guessing they are necessary :-)

**Review by chriskmnds — APPROVED** [review]
Confirmed as per testing instructions. LGTM

> Go to /earn/[site]?flags=plans/pro-plan on a Pro site
> The "Collect PayPal payments" and "Earn ad revenue" should be unlocked

This is the same without the query string (`?flags=plans/pro-plan`)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7212193

Thank you @claudiucelfilip for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7212193).

**Pull request merged**